shell.nix

```diff
@@ -17,7 +17,7 @@ pkgs.mkShell {
 
   buildInputs = with pkgs; [
     pkg-config
-    rust-bin.stable."1.80.0".default
+    rust-bin.stable."1.85.1".default
```

Comment:
For this version to be accessible in here, we need to update the overlay commit hash to the latest (few lines above):
```
overlays = [
      (import (fetchGit {
        url = "https://github.com/oxalica/rust-overlay.git";
        rev = "a0e3395c63cdbc9c1ec17915f8328c077c79c4a1";
      }))
 ]
```
I think comment here would also help.

Reply:
Is my understanding correct? https://github.com/logos-co/nomos/pull/1191/commits/ec95642864e62b8348edeb7cf7e482417540754c

Reply:
> Is my understanding correct?

Yes!